test/integration/scripts/run-test-suites.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash -v
+
+set -e
+
+auth0 login \
+   --domain "${AUTH0_CLI_CLIENT_DOMAIN}" \
+   --client-id "${AUTH0_CLI_CLIENT_ID}" \
+   --client-secret "${AUTH0_CLI_CLIENT_SECRET}"
+
+set +e
+
+commander test --filter "$FILTER" --dir ./test/integration
```

Comment:
Small n+1 over here. By extracting this logic outside of the makefile, we make these steps a bit more readable and also set us up for splitting up the test-case.yml into multiple suites for each command.